streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -37,23 +37,23 @@
   private List<ExecutionJobEdge> inputEdges = new ArrayList<>();
   private List<ExecutionJobEdge> outputEdges = new ArrayList<>();
 
-  public ExecutionJobVertex(JobVertex jobVertex) {
+  public ExecutionJobVertex(JobVertex jobVertex, Map<String, String> jobConfig) {
```

Comment:
Why  `ExecutionJobVertex ` need `jobConfig`?  I thought it only need vertex related config

Reply:
Because ```jobconf``` contains the resource configuration required for task execution